coderd/database/migrations/000219_notifications.up.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TYPE notification_message_status AS ENUM (
+    'pending',
+    'leased',
+    'sent',
+    'permanent_failure',
+    'temporary_failure',
+    'unknown'
+    );
+
+CREATE TYPE notification_method AS ENUM (
+    'smtp',
+    'webhook'
+    );
+
+CREATE TABLE notification_templates
+(
+    id             uuid                 NOT NULL,
+    name           text                 NOT NULL,
+    enabled        boolean DEFAULT TRUE NOT NULL,
+    title_template text                 NOT NULL,
+    body_template  text                 NOT NULL,
+    actions        jsonb,
```

Comment:
Got it now. The upstack PR does use it, I can see. I didn't find it because I was looking at the main `NotificationTemplate` type, and it's used in `FetchNewMessageMetadataRow`